### ISSUE TITLE
TypeError: setting getter-only property "children" on press "Add item" in any ::REPEATER in Elementor Editor

### ISSUE THREAD

**Issue by VadymPogorelov** [elementor/elementor#15665]
When using a field of type REPEATER, clicking on the "Add item" button causes a js error 
TypeError: setting getter-only property "children"

See screenshot: https://prnt.sc/1bg5e8r

**Comment by VadymPogorelov** [comment]
Conducted the following experiment: 

1. Added the output to the console: https://prnt.sc/1bqhmf0
2. I create a new section and while it is empty - press the button "Add item" in the field with type 'REPEATER" - everything is OK: a new group of fields is added, there are no errors in the console: https://prnt.sc/1bqjsxx
3. I add any module to one of the columns, for example, a button: https://prnt.sc/1bqkblh
4. Select the section again and press the button "Add item" in the field with type 'REPEATER" - error messages appear. Moreover, according to the output to the console, there is a problem with the property "children" of the added element "button": https://prnt.sc/1bqml8s
5. At the same time, if you select the "button" element and set the same field in its parameters, everything is ok: https://prnt.sc/1bqnlhe

What could be the problem?

**Comment by VadymPogorelov** [comment]
I continued experimenting and this is what I found out: 

The error occurs due to the fact that a field of the REPEATER type has been added to each module (for selecting sound effects for mouse events): https://prnt.sc/1brqseo.
If I remove this field from all modules https://prnt.sc/1brwodi, the error disappears and everything works. But I need this field to sound the visitor's actions!

And apparently the Elementor widgets are used the so-called "lazy" getter for the property "children", which initializes this property to an empty array on the first call: https://prnt.sc/1bru9on
Therefore, it may not be necessary to assign `this.children = [];`
I commented out this line `this.children = [];` https://prnt.sc/1brvabo
And it all worked

I do not offer this as a solution, I just point out that if a section has a field of type REPEATER and modules nested in the section also have such a field, then when you click on Add Item in the section, a js error occurs

**Comment by nerds-farm** [comment]
I confirm the problem, Widgets are not affected, only Sections and Columns,
I analyze the Editor js and the error appear in this lines (10509):

onButtonAddRowClick: function onButtonAddRowClick() {
    var newModel = $e.run('document/repeater/insert', {
      container: this.options.container,
      name: this.model.get('name'),
      model: this.getDefaults()
    });
    ...

variable newModel is undefined, so fail on next row.

**Comment by mohon-xs** [comment]
Yes, it's the same issue I'm facing, even in the latest production version.

**Comment by easin-xs** [comment]
Yes, I am facing the same issues in the latest version. Please fix it.

**Comment by VadymPogorelov** [comment]
The update 3.4.2 fixed the problem. 

But this situation has already arisen earlier - this problem appeared, then after some update it disappeared, but after a while it appeared again.
